.travis.yml

```diff
@@ -33,6 +34,7 @@ script: vendor/bin/phpunit -c phpunit.xml.dist --no-coverage --colors --columns
 
 jobs:
   allow_failures:
+    - php: 7.3
```

Comment:
Probably should remove this from the `allow_failures` now, since PHP 7.3 is stable (and looks like the tests pass fine against it).

You may want to add `php: 7.4snapshot` to the matrix (and the `allow_failures`) since that's available on travis now.

Reply:
updates